crates/vim/src/command.rs

```diff
@@ -604,6 +604,8 @@ fn generate_commands(_: &AppContext) -> Vec<VimCommand> {
         VimCommand::new(("$", ""), EndOfDocument),
         VimCommand::new(("%", ""), EndOfDocument),
         VimCommand::new(("0", ""), StartOfDocument),
+        // TODO: vim prints "E37: No write since last change (add ! to override)" if you have unsaved changes and you run :edit without the bang
+        VimCommand::new(("e", "dit"), editor::actions::ReloadFile).bang(editor::actions::ReloadFile),
```

Comment:
looking for suggestions on how to handle this. in vim, `:edit` will not throw away your changes, but `:edit!` will. to quote https://vimhelp.org/editing.txt.html#:edit

```
:e[dit] [++opt] [+cmd]	Edit the current file.  This is useful to re-edit the
			current file, when it has been changed outside of Vim.
			This fails when changes have been made to the current
			buffer and 'autowriteall' isn't set or the file can't
			be written.
			Also see ++opt and +cmd.

							:edit! discard
:e[dit]! [++opt] [+cmd]
			Edit the current file always.  Discard any changes to
			the current buffer.  This is useful if you want to
			start all over again.
			Also see ++opt and +cmd.

							:edit_f
```

i don't think i ever run :e without any arguments in vim, always `:e!`, is there a way to only map `:e!` `:edit!` in zed? or will that require changes to VimCommand ?

Reply:
We can either keep them the same (as you have done), and improve it later; or we could add another action that is just a no-op. I think it'd be better to keep them the same (until we add argument support and we get another try).